sympy/vector/implicitregion.py

```diff
@@ -76,37 +77,165 @@ def regular_point(self):
 
         >>> from sympy.abc import x, y
         >>> from sympy.vector import ImplicitRegion
-        >>> circle = ImplicitRegion((x, y), (x + 2)**2 + (y - 3)**2 - 16)
+        >>> circle = ImplicitRegion((x, y), x**2 + y**2 - 4)
         >>> circle.regular_point()
-        (-6, 3)
+        (2, 0)
+        >>> parabola = ImplicitRegion((x, y), x**2 - 4*y)
+        >>> parabola.regular_point()
+        (0, 0)
+        >>> r = ImplicitRegion((x, y, z), (x + y + z)**4)
+        >>> r.regular_point()
+        (-10, -10, 20)
 
-        """
+        Refrences
+        =========
+
+        - Erik Hillgarter, Rational Points on Conics.
```

Comment:
Publisher and year are usually mentioned when citing research.

Reply:
OK